catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java

```diff
@@ -483,6 +486,14 @@ public boolean saveFormulaIfPossible() {
 		InternFormulaParser formulaToParse = formulaEditorEditText.getFormulaParser();
 		FormulaElement formulaParseTree = formulaToParse.parseFormula();
 
+		MenuItem undo = currentMenu.findItem(R.id.menu_undo);
+		undo.setIcon(R.drawable.icon_undo_disabled);
+		undo.setEnabled(false);
+
+		MenuItem redo = currentMenu.findItem(R.id.menu_redo);
+		redo.setIcon(R.drawable.icon_redo_disabled);
+		redo.setEnabled(false);
+
```

Comment:
I would put this code on line [473](https://github.com/Catrobat/Catroid/pull/1294/files#diff-d5336e4f047445d9085f6c015edeb69eR473) in method `setInputFormula()`. It has nothing to do with `saveFormulaIfPossible()`.